config-connector/solutions/sql/kpt/mysql-ha/README.md

```diff
@@ -0,0 +1,32 @@
+MySQL High Availability
+==================================================
+# NAME
+  mysql-ha
+# SYNOPSIS
+  Config Connector compatible YAMLs for creating a high availability MySQL cluster
+# CONSUMPTION
+  Download the package using [kpt](https://googlecontainertools.github.io/kpt/).
+  ```
+  kpt pkg get https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/config-connector/solutions/sql/kpt/mysql-ha mysql-ha
+  ```
+# USAGE
+## SETTERS
+|     NAME      |       VALUE       |     SET BY      |       DESCRIPTION       | COUNT |
+|---------------|-------------------|-----------------|-------------------------|-------|
+| database-name | mysql-ha-database | package-default | name of SQL database    | 1     |
+| instance-name | mysql-ha-instance | package-default | name of SQL instance    | 5     |
+| password      | ${PASSWORD?}      | PLACEHOLDER     | password of SQL user    | 1     |
+| region        | us-central1       | package-default | region of SQL instance  | 2     |
+| user-name     | mysql-ha-user     | package-default | name of SQL user        | 1     |
```

Comment:
Can we make a user name a placeholder, which will encourage the users to come up with their own user name and password, rather than encourage spreading `mysql-ha-user`?